app/controllers/v0/vic/profile_photo_attachments_controller.rb

```diff
@@ -18,8 +23,20 @@ def create
       end
 
       def show
-        form_attachment = ::VIC::ProfilePhotoAttachment.where(guid: params[:id]).first
-        render(json: form_attachment)
+        guid = params[:id]
+
+        raise ActiveRecord::RecordNotFound unless GUID_PATTERN.match(guid)
+
+        form_attachment = ::VIC::ProfilePhotoAttachment.where(guid: guid).first
+        file = form_attachment.get_file
+
+        begin
+          headers['Content-Type'] = file.content_type
+          headers['Content-Disposition'] = "inline; filename=\"#{file.filename}\""
+          file.read { |c| response.stream.write(c) }
```

Comment:
how about using send_data?
https://apidock.com/rails/ActionController/DataStreaming/send_data

Reply:
Was using send_data originally, but we'd still be buffering the `file.read` locally first. Patrick pointed me to how they solved that similarly here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/health_record_contents_controller.rb